dashboard/src2/pages/LoginSignup.vue

```diff
@@ -588,9 +761,9 @@ export default {
 				return 'Reset password';
 			} else if (this.isLogin) {
 				if (this.saasProduct) {
-					return `Sign in to your account to start using ${this.saasProduct.title}`;
+					return `Log In to your account to start using ${this.saasProduct.title}`;
 				}
-				return 'Sign in to your account';
+				return 'Log In to your account';
```

Comment:
Any specific reason why we are changing the wording here? Also why is "In" capitalized in the middle of a sentence.

Reply:
Rest of the texts says log in in most places (like button), thought we could change it to make it uniform 🤔.

Reply:
Fixed the casing